rma_account/wizards/rma_add_invoice.xml

```diff
@@ -14,7 +14,7 @@
                 </group>
                 <separator string="Select Invoices lines to add"/>
                 <field name="invoice_line_ids"
-                       domain="[('invoice_id.partner_id', '=', partner_id),
+                       domain="[('invoice_id.commercial_partner_id', '=', partner_id),
```

Comment:
Thanks for this Pull Request!

Perhaps here we can try here with:

`domain="[('invoice_id.commercial_partner_id', 'child_of', partner_id),`

That way if will take both, the commercial_partner, the partner and all the children. Do you agree?

Reply:
Ok, I do that.

Reply:
Thank you. I will test if that woks before merging. Does it work on your side?

Reply:
Yes, it works on my side